components/App.jsx

```diff
@@ -1,5 +1,5 @@
 // the worms control the spice 
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
```

Comment:
It looks like useEffect isn't used, so I would remove this

Reply:
Or better yet, I might define location inside of a useEffect. It looked like others may have been doing that, and its probably healthier for the component / state control